Broken TypeScript definitions

`npm install --save-dev "@types/bluebird"`

```
node_modules/filehound/index.d.ts:2:8 - error TS1259: Module '"./node_modules/@types/bluebird/index"' can only be default-imported using the 'esModuleInterop' flag

2 import Bluebird from "bluebird";
         ~~~~~~~~

  node_modules/@types/bluebird/index.d.ts:1262:1
    1262 export = Bluebird;
         ~~~~~~~~~~~~~~~~~~
    This module is declared with using 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.

node_modules/filehound/index.d.ts:108:31 - error TS7051: Parameter has a name but no type. Did you mean 'arg0: any'?

108     public addFilter(filter: (...any) => boolean): FileHound;
                                  ~~~~~~
```

`npm remove --save-dev "@types/bluebird"`

```
node_modules/filehound/index.d.ts:2:22 - error TS7016: Could not find a declaration file for module 'bluebird'. './node_modules/bluebird/js/release/bluebird.js' implicitly has an 'any' type.
  Try `npm install @types/bluebird` if it exists or add a new declaration (.d.ts) file containing `declare module 'bluebird';`

2 import Bluebird from "bluebird";
                       ~~~~~~~~~~

node_modules/filehound/index.d.ts:108:31 - error TS7051: Parameter has a name but no type. Did you mean 'arg0: any'?

108     public addFilter(filter: (...any) => boolean): FileHound;
                                  ~~~~~~
```

--- a/index.d.ts
+++ b/index.d.ts
@@ -1,5 +1,5 @@
 import { EventEmitter } from "events";
-import Bluebird from "bluebird";
+import * as Bluebird from "bluebird";
 
 declare module 'filehound' {
   class FileHound extends EventEmitter {

